Cargo.lock

```diff
@@ -6637,9 +6637,9 @@ checksum = "589f6da84c646204747d1270a2a5661ea66ed1cced2631d546fdfb155959f9ec"
 
 [[package]]
 name = "winit"
-version = "0.30.9"
+version = "0.30.10"
 source = "registry+https://github.com/rust-lang/crates.io-index"
```

Comment:
Oh, I see, the lockfile was already at `.9`, only the requirement stayed behind at `.8`...